### PR TITLE
Mothington city update

### PR DESCRIPTION
**Pull request by Gr33d-y**
Changes moths so they can use their wings outside in space.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
 		TRAIT_CAN_USE_FLIGHT_POTION,
 		TRAIT_TACKLING_WINGED_ATTACKER,
 		TRAIT_ANTENNAE,
+		TRAIT_SPACEWALK,
```
[the remark this answers is not in the document]
Comment (Gr33d-y) [reply code/modules/mob/living/carbon/human/species_types/mothmen.dm:13]: Disagree, all races should, and will, have special things about them so they do not feel like "humans but take more burn damage" That goes for all of them besides baseline races, who will have no downsides but very small things to make them feel more flavorful. That being said, if this does show to be more powerful than it should be, we can change moths more to make them have a larger downside.

[the remark this answers is not in the document]
Comment (Gr33d-y) [reply code/modules/mob/living/carbon/human/species_types/mothmen.dm:13]: Just as a reminder, this is a patch to make all races feel flavorful and allow them to do great things that other races can not do. A moth might look at a teshari and wish they could move as fast as them, but a teshari will look at a moth and wish they could fly in space. This also opens up opportunities for RP, moths can go on little space flights with each other. Keep in mind, this does NOT make them space proof, it only allows them to fly in space.

[the remark this answers is not in the document]
Comment (mcmeiler) [reply code/modules/mob/living/carbon/human/species_types/mothmen.dm:13]: Ooooo, this makes moths ALWAYS be able to move freely in space. This doesn't make it tied to wings.

[the remark this answers is not in the document]
Comment (Gr33d-y) [reply code/modules/mob/living/carbon/human/species_types/mothmen.dm:13]: I would love for it to be tied to the wings, and that is the plan, but for now we will have this.

**Comment by Gr33d-y** [comment]
Closing this as I've realized that to do justice to most species I would need to be able to code far better than I can currently.

**Pull request closed without merging**